# Question 1

Data description α== 0.05(95% Confidence) Y == Continious X == Discrete is Y1,Y2 are normal?

In [2]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

In [3]:
#Load the dataset
data = pd.read_csv('C:\\Users\\amind\\Downloads\\Cutlets.csv')
data.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

In [4]:
UnitA = pd.Series(data.iloc[:,0])
UnitA

0     6.8090
1     6.4376
2     6.9157
3     7.3012
4     7.4488
5     7.3871
6     6.8755
7     7.0621
8     6.6840
9     6.8236
10    7.3930
11    7.5169
12    6.9246
13    6.9256
14    6.5797
15    6.8394
16    6.5970
17    7.2705
18    7.2828
19    7.3495
20    6.9438
21    7.1560
22    6.5341
23    7.2854
24    6.9952
25    6.8568
26    7.2163
27    6.6801
28    6.9431
29    7.0852
30    6.7794
31    7.2783
32    7.1561
33    7.3943
34    6.9405
Name: Unit A, dtype: float64

In [5]:
UnitB = pd.Series(data.iloc[:,1])
UnitB

0     6.7703
1     7.5093
2     6.7300
3     6.7878
4     7.1522
5     6.8110
6     7.2212
7     6.6606
8     7.2402
9     7.0503
10    6.8810
11    7.4059
12    6.7652
13    6.0380
14    7.1581
15    7.0240
16    6.6672
17    7.4314
18    7.3070
19    6.7478
20    6.8889
21    7.4220
22    6.5217
23    7.1688
24    6.7594
25    6.9399
26    7.0133
27    6.9182
28    6.3346
29    7.5459
30    7.0992
31    7.1180
32    6.6965
33    6.5780
34    7.3875
Name: Unit B, dtype: float64

In [6]:
# 2-sample 2-tail ttest: 
p_value = stats.ttest_ind(UnitA,UnitB)
p_value

Ttest_indResult(statistic=0.7228688704678063, pvalue=0.4722394724599501)

In [7]:
# 2 tail probability
p_value[1]

0.4722394724599501

 p value of the Paired T test is == 0.45 > α  
 Thus H0 is accepted
 Mean of Y1 and Y2 are normal

# Question 2

Data description
α== 0.05(95% Confidence)
Y == Continious
X == Discrete
is Y1,Y2,Y3 and Y4 are normal?

In [8]:
df = pd.read_csv('C:\\Users\\amind\\Downloads\\LabTAT.csv')

In [9]:
# Normality test
stats.shapiro(df['Laboratory 1'])

ShapiroResult(statistic=0.990183413028717, pvalue=0.5507769584655762)

In [10]:
stats.shapiro(df['Laboratory 2'])

ShapiroResult(statistic=0.9936320185661316, pvalue=0.8637352585792542)

In [11]:
stats.shapiro(df['Laboratory 3'])

ShapiroResult(statistic=0.988633930683136, pvalue=0.42045730352401733)

In [12]:
stats.shapiro(df['Laboratory 4'])

ShapiroResult(statistic=0.9913760423660278, pvalue=0.6619619131088257)

H0 is accepted . That is Y1,Y2,Y3 and Y4 are normal

In [13]:
#Varience
stats.levene(df['Laboratory 1'], df['Laboratory 2'], df['Laboratory 3'], df['Laboratory 4'])

LeveneResult(statistic=2.599642500418024, pvalue=0.05161343808309816)

P value of variance test is == 0.051 > α
H0 is accepted

In [15]:
stats.f_oneway(df['Laboratory 1'], df['Laboratory 2'], df['Laboratory 3'], df['Laboratory 4'])

F_onewayResult(statistic=118.70421654401437, pvalue=2.1156708949992414e-57)

P value of the One way Anova test is == 2e-16 < α
Thus H1 is accepted.
Mean TAT for 4 laboratories notb equal (There is a signifcance difference between TAT of the laboratories)

# Question 3

In [1]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

Data description α== 0.05(95% Confidence) Y == Discrete X == Discrete 
Since there are more than 2 variable we will perform  Chi-square test

In [2]:
#Chi-Square test
df = pd.read_csv('C:\\Users\\amind\\Downloads\\BuyerRatio.csv')

In [3]:
stats.chi2_contingency([df['East'],df['West'],df['North'],df['South']])

(1.5959455386610577,
 0.6603094907091882,
 3,
 array([[  42.76531299,  442.23468701],
        [ 146.81287862, 1518.18712138],
        [ 131.11756787, 1355.88243213],
        [  72.30424052,  747.69575948]]))

P value of Chi-Square test == 0.66 > α
Thus H0 is accepted
The male-female buyer rations are smilar across regions.

# Question 4

Data description α== 0.05(95% Confidence) Y == Discrete X == Discrete 
Since there are more than 2 variable we will perform  Chi-square test

In [4]:
#Chi-Square test
df = pd.read_csv('C:\\Users\\amind\\Downloads\\Costomer+OrderForm.csv')

In [7]:
stats.chi2_contingency([df['Phillippines'].value_counts(),df['Indonesia'].value_counts(),df['Malta'].value_counts(),
                        df['India'].value_counts()])

(3.8589606858203545,
 0.2771020991233144,
 3,
 array([[271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25],
        [271.75,  28.25]]))

P value of Chi-Square test == 0.277 > α
Thus H0 is accepted
The defective % does not varies significantly by centres